.github/qodo.yml

```
@@ -0,0 +1,30 @@
+        with:
+          node-version: '20'
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - name: Generate tests with QODO
+        run: |
+          # QODO will auto-detect Vitest and generate co-located tests
+          npx qodo-cli test generate --framework=vitest --location=colocated
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)